miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultPoolOrderService.java

```diff
@@ -116,10 +111,4 @@ public void beforeDelete(PoolOrder object) throws IOException {
     poolService.save(pool);
   }
 
-  @Override
-  public void authorizeDeletion(PoolOrder object) throws IOException {
-    Pool pool = poolService.get(object.getPool().getId());
-    authorizationManager.throwIfNotWritable(pool);
```

Comment:
Removing this means that only admins will be able to delete orders. Please update user manual to indicate this

Reply:
Hmm. Should we replace that with a admin-or-creator?

Reply:
Probably. "Writeable" technically meant everyone before, but I don't think everyone should be able to delete orders

Reply:
Sounds good to me.